__tests__/__snapshots__/colors.test.js.snap

```diff
@@ -372,6 +372,359 @@ exports[`renders Polaris.sketchpalette correctly 1`] = `
 "
 `;
 
+exports[`renders colors.ase.json correctly 1`] = `
+"{
+  \\"version\\": \\"1.0\\",
+  \\"groups\\": [],
+  \\"colors\\": [
+    {
+      \\"name\\": \\"purple-lighter\\",
+      \\"model\\": \\"RGB\\",
+      \\"type\\": \\"global\\",
+      \\"color\\": [0.9647058823529412, 0.9411764705882353, 0.9921568627450981]
+    },
+    {
+      \\"name\\": \\"purple-light\\",
+      \\"model\\": \\"RGB\\",
+      \\"type\\": \\"global\\",
+      \\"color\\": [0.8901960784313725, 0.8156862745098039, 1]
+    },
+    {
+      \\"name\\": \\"purple-base\\",
```

Comment:
Curious, can we get away without the base suffix? Or do you think it's better to be consistent and have all colors have a suffix?

Reply:
You're not the first to ask :) See #16

Reply:
Hah, would be a nice to have for us but not necessary here.